src/main/java/aquality/selenium/browser/Browser.java

```diff
@@ -42,7 +42,7 @@ public Browser(RemoteWebDriver remoteWebDriver) {
         this.timeouts = AqualityServices.get(ITimeoutConfiguration.class);
         webDriver = remoteWebDriver;
         this.implicitTimeout = timeouts.getImplicit();
-        getDriver().manage().timeouts().implicitlyWait(implicitTimeout.getSeconds(), TimeUnit.SECONDS);
+        getDriver().manage().timeouts().implicitlyWait(Duration.ofSeconds(implicitTimeout.getSeconds()));
```

Comment:
implicitTimeout already has a type Duration. Why do you want to do such convertion `Duration.ofSeconds(duration.getSeconds)` instead of just passing the duration object?

Reply:
Fixed